op-chain-ops/cmd/celo-migrate/README.md

```diff
@@ -0,0 +1,3 @@
+# Celo L1 -> Cel2 migration tool
+
+This tool allows migrating the state of a Celo chain to a state usable in op-geth.
```

Comment:
I wonder if such a small README is helpful, the information here is also contained in the help text of the tool. I imagine most uses opening this file only to find it tells them nothing more than they already knew.